File: src/Player.java

class Player {
	private int tokens_put;
	private int tokens_left;
	private String name;
	private String color;
	
	public Player(String name, String color) {
		this.tokens_left = 0;
		this.tokens_put = 0;
		this.name = name;
		this.color = color;
	}
	
	public int getTokensPut( ){
		return tokens_put;
	}
	
	public int getTokensLeft( ){
		return tokens_left;
	}
	
	public String toString() {
		String temp = String.format("%s %s", this.name, this.color);
		return temp;
		
	}
	
	public void putToken() {
		this.tokens_put += 1;
		this.tokens_left += 1;
	}
	
	public void removeToken() {
		this.tokens_left -= 1;
	}
}

File: morris/morris_game/Player.py
'''
Created on Oct 3, 2015

@author: speng
'''

class Player(object):

    def __init__(self, name, color):
        '''
        Constructor
        '''
        self.tokens_put = 0
        self.tokens_left = 0   
        self.name = name
        self.color = color
    
    def __str__(self, *args, **kwargs):
        return "{player} {color}".format(player = self.name, color = self.color) 
    
    def put_token(self):
        self.tokens_put = self.tokens_put +1
        self.tokens_left = self.tokens_left +1
        
    def remove_token(self):
        self.tokens_left = self.tokens_left -1